lib/Ohmbrewer_Screen.cpp

```diff
@@ -360,3 +362,97 @@ unsigned long Ohmbrewer::Screen::displayStatusUpdate(char *statusUpdate) {
 
     return micros() - start;
 }
+
+/**
+ * Checks for a touch event and triggers actions 
+ *  if the the touch was on a screen "button".
+ * @returns Time it took to run the function
+ */
+unsigned long Ohmbrewer::Screen::captureButtonPress() {
+    unsigned long start = micros();
+    char printx [10];
+	char printy [10];
+	char status [40];
+	
+	// a point object holds x y and z coordinates
+  	TSPoint p = ts.getPoint();
+  	//According to Particle forums, the read below is necessary to get touch to work on Photon
+  	ts.readTouchY();
```

Comment:
Do we only need to do this once after boot-up, or once every single time we're scanning for a touch? If it's the former, maybe this can be moved into the Screen constructor?